modules/kernel/src/main/scala/tofu/Clock.scala

```diff
@@ -0,0 +1,67 @@
+package tofu
+
+import tofu.internal.EffectComp
+import tofu.internal.carriers._
+
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.FiniteDuration
+
+trait Clock[F[_]] {
+
+  /** Returns the current time, as a Unix timestamp (number of time units
+    * since the Unix epoch), suspended in `F[_]`.
+    *
+    * This is the pure equivalent of Java's `System.currentTimeMillis`,
+    * or of `CLOCK_REALTIME` from Linux's `clock_gettime()`.
+    */
+  def realTime(unit: TimeUnit): F[Long]
```

Comment:
If tofu will have its own Clock typeclass, so why should it be exactly like cats one? Is it possible to fix some of the design problems e.g. not having information on units of measurement in the signature (ce3 style with FiniteDuration for example (but rather something better))?

Reply:
You have some specific design in mind?